internal/driver/nodeserver.go

```diff
@@ -376,20 +376,7 @@ func (ns *NodeServer) NodeGetInfo(ctx context.Context, req *csi.NodeGetInfoReque
 
 	log.V(2).Info("Processing request", "req", req)
 
-	// Get the number of currently attached instance disks, and subtract it
-	// from the limit of block devices that can be attached to the instance,
-	// which will effectively give us the number of block storage volumes
-	// that can be attached to this node/instance.
-	//
-	// This is what the spec wants us to report: the actual number of volumes
-	// that can be attached, and not the theoretical maximum number of
-	// devices that can be attached.
-	log.V(4).Info("Listing instance disks", "nodeID", ns.metadata.ID)
-	disks, err := ns.client.ListInstanceDisks(ctx, ns.metadata.ID, nil)
-	if err != nil {
-		return &csi.NodeGetInfoResponse{}, errInternal("list instance disks: %v", err)
-	}
-	maxVolumes := maxVolumeAttachments(ns.metadata.Memory) - len(disks)
+	maxVolumes := maxVolumeAttachments(ns.metadata.Memory)
```

Comment:
The above comment seems to conflict with this change, do you have documentation for this being the correct approach?